tooling/nargo_fmt/tests/expected/nested-if-else.nr

```diff
@@ -0,0 +1,11 @@
+fn nested_if_else() {
+    if false {
+    1
+} else {
```

Comment:
Some indentation to improve upon here as well. Since this PR is still a draft, I'm assuming this is already known and I'll stop pointing it out.